client/src/main/scala/org/apache/celeborn/client/ApplicationHeartbeater.scala

```diff
@@ -48,13 +48,14 @@ class ApplicationHeartbeater(
         override def run(): Unit = {
           try {
             require(rssHARetryClient != null, "When sending a heartbeat, client shouldn't be null.")
-            val (tmpTotalWritten, tmpFileCount) = shuffleMetrics()
-            logDebug(s"Send app heartbeat with $tmpTotalWritten $tmpFileCount")
+            val (tmpTotalWritten, tmpTotalFileCount) = shuffleMetrics()
+            logInfo("Send app heartbeat with " +
```

Comment:
I suppose print it in each 10s is not noisy

Reply:
It's fine to me, though I think for long job it will print pretty much~